askama_derive/src/generator/filter.rs

```diff
@@ -643,6 +706,8 @@ fn ensure_filter_has_feature_alloc(
     Ok(())
 }
 
+const FALSE: &WithSpan<'static, Expr<'static>> = &WithSpan::new_without_span(Expr::BoolLit(false));
```

Comment:
Considering it's only used by `visit_default_filter`, maybe it'd be better to move it there?

Reply:
It's also used in `visit_indent_filter`.